utils/s3/retry_strategy.cc

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include "retry_strategy.hh"
+#include "aws_error.hh"
+#include "log.hh"
+
```

Comment:
could you please s|log.hh|utils.log.hh| ? sorry for introducing this breaking change in 6ead5a4696636f00fa673f4772dcb272eab2948e .

Reply:
done